Assets/Scenes/scripts/UI/DialogueManager.cs

```diff
@@ -0,0 +1,113 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DialogueManager : MonoBehaviour
+{
+    public Image textBox;
+    public TextMeshProUGUI text;
+
+    public string[] sentences;
+    string currentText;
+    int index;
```

Comment:
Ambiguous name. Something like "curSentence" or "sentenceIndex" would be more descriptive.